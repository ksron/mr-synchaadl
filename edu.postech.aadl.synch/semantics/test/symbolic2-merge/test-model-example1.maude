
set show timing off .

load ../../interpreter-symbolic2-merge.maude
load ../symbolic/module-synchronous-model.maude

mod TEST-MODEL-TRANSITION-SYSTEM is
  including TEST-MODEL-TRANSITION-SYSTEM-MOD .

  eq @m@ = ['TEST-MODEL-TRANSITION-SYSTEM-MOD] .
endm


search [,50] {[true] || obj4 | 0 | infty} =>+ C:GlobalSystem .
