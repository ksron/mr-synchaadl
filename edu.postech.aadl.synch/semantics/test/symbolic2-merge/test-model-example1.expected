==========================================
search [, 50] in TEST-MODEL-TRANSITION-SYSTEM : {[(true).Boolean] || obj4 | 0 |
    infty} =>+ C:GlobalSystem .

No solution.
states: 1  rewrites: 335
Bye.
